README.md

```diff
@@ -7,7 +7,7 @@ The simplest way to install nVidia drivers on macOS.
 Simply copy and paste the following in a terminal:
 
 ```
-bash <(curl -s https://raw.githubusercontent.com/Benjamin-Dobell/nvidia-update/master/nvidia-update.sh)
+curl -s https://raw.githubusercontent.com/Benjamin-Dobell/nvidia-update/master/nvidia-update.sh | bash
```

Comment:
As mentioned in #9, this change is unfortunately incorrect - explanatory links regarding process substitution included.

Reply:
Updated. Hope that it helps.